x-pack/packages/security-solution/ecs_data_quality_dashboard/impl/data_quality/data_quality_panel/index_properties/helpers.test.ts

```diff
@@ -54,30 +39,27 @@ describe('helpers', () => {
     });
 
     describe('when `mappingsProperties` is unknown', () => {
+      const incompatibleFieldMetadata = {
```

Comment:
@angorayc I think current naming is correct because when `mappingProperies` is `null` or `undefined` the idea is to return an incompatible field to show it as a result of a check. You can see in the test title further down below that incompatible is used many times on purpose. It's also true that we add this into the collection of incompatible fieldMetadata explicitly when returning partitionedMetadata.

For example If I create an empty index without any mapping and try to check it I get back an incompatible field as a result of check denoting that `@timestamp` is invalid, as in - it's required but not present and it should be.

![image](https://github.com/elastic/kibana/assets/1625373/df5e8033-3098-4a2f-a4e4-890b0570a2f7)

Reply:
I see, actually in this scenario it's valid that the `all` property contains the same data as `incompatible`, and it's indeed incompatibleFieldMetadata. Got it, I'm fine with it, thank you!